platforms/platform-googleassistant/src/GoogleAssistantPlatform.ts

```diff
@@ -87,14 +87,11 @@ export class GoogleAssistantPlatform extends Platform<
       Object.keys(request.intent?.params || {}).length &&
       request.session?.params?._GOOGLE_ASSISTANT_SELECTION_INTENT_
     ) {
-      if (!jovo.$nlu) {
-        jovo.$nlu = {};
-      }
-      if (!jovo.$nlu.intent) {
-        jovo.$nlu.intent = { name: '' };
+      if(!jovo.$input.nlu) {
+        jovo.$input.nlu = {};
       }
       // set intent
-      jovo.$nlu.intent.name = request.session.params._GOOGLE_ASSISTANT_SELECTION_INTENT_;
+      jovo.$input.nlu.intent = request.session.params._GOOGLE_ASSISTANT_SELECTION_INTENT_
```

Comment:
`$input.nlu` should only be created by NLU integrations. Since this is a mapping that happens without an NLU, I think we should add this to the root of `$input`.

Btw, this should be the same for internal mapping for `selection` of Alexa APL as well. Let's talk about this tomorrow.